### PR TITLE
Vitaly/feature/multiply export

### PR DESCRIPTION
**Pull request by Ziles131**
Vitaly/feature/multiply export
1. Create_pdf_function
2. Add multiply_export logic to the biddings page
3. Remove comments

### CODE REVIEW

File: app/controllers/create_pdf_file.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import os
+import io
+
+from flask import render_template
+
+from .price import calculate_subtotal
+
+from app.models import Bid, LinkWorkItem, WorkItemGroup
+import pdfkit
```
Comment (kipod) [inline app/controllers/create_pdf_file.py]: incorrect import sequence

File: app/controllers/create_pdf_file.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+def create_pdf_file(bid_id):
+    bid = Bid.query.get(bid_id)
+    global_work_items = (
+        LinkWorkItem.query.filter(LinkWorkItem.bid_id == bid_id)
+        .filter(LinkWorkItem.work_item_group == None)  # noqa 711
+        .all()
+    )
+    groups = WorkItemGroup.query.filter(WorkItemGroup.bid_id == bid_id).all()
+    BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+    PATH_TO_IMG = os.path.join(BASE_DIR, "static/images/")
```
Comment (kipod) [inline app/controllers/create_pdf_file.py]: move them to global variables

File: app/controllers/price.py [hunk 3]
```diff
@@ -1,8 +1,9 @@
 from flask import current_app as app
 from app.models import Bid
+from sqlalchemy import inspect
```
Comment (kipod) [inline app/controllers/price.py]:
```python
from flask import current_app as app
from sqlalchemy import inspect
from app.models import Bid
```

File: app/static/js/bidding.js [hunk 4]
```diff
@@ -58,3 +63,52 @@ clientCloseWrapper.addEventListener('click', (e) => {
...
   };
 });
+
+// TBD Choice
+
+const inputs = document.querySelectorAll('input[type="checkbox"]');
+const bidID = document.querySelector('.bidIdJs').getAttribute('value');
+inputs.forEach( el => {
+  const myResponse = async () => {
+    const response = await fetch(`http://127.0.0.1:5000/check_tbd/${bidID}/${el.getAttribute('name')}`, {method: 'GET'})
```
Comment (kipod) [inline app/static/js/bidding.js]: http://127.0.0.1:5000 ??? are you sure?

**Review by kipod — CHANGES_REQUESTED** [review]

File: app/controllers/create_pdf_file.py (hunk 1 above)
Comment (Ziles131) [reply app/controllers/create_pdf_file.py]: pdfkit it's not a standard library, is it should be above?
like that:
```python
import pdfkit
from .price import calculate_subtotal
from app.models import Bid, LinkWorkItem, WorkItemGroup
```

**Pull request merged**